src/bioclip_vector_db/vector_db.py

```diff
@@ -239,7 +241,7 @@ def main():
 
     # Currently only CHROMA backend is supported so hardcoding is fine.
```

Comment:
Could we add another CLI argument, e.g. `--storage`, to configure this now that lists the enum options available?